### ISSUE TITLE
io.read() on stdin?

### ISSUE THREAD

**Issue by LEGOlord208** [Shopify/go-lua#77]
Am I stupid - Or is `io.read()` broken?  
Attempt:
```Lua
thing = io.read();
exec("say User input: "..thing);
```

I've also tried `read("*line")` (which gave a nil result) and `readLine()` (method not found)

What do I do?  
Making my own reading method works fine, but...

**Comment by fbogsany** [comment]
Yes: https://github.com/Shopify/go-lua/blob/master/io.go#L120-L125

**Comment by legolord208** [comment]
Oh, it's `// TODO`d :P
